{
    "id": "CVE-2019-10212",
    "product": "undertow",
    "version": "n/a",
    "vulnerability": [
        "CWE-532"
    ],
    "description": "A flaw was found in, all under 2.0.20, in the Undertow DEBUG log for io.undertow.request.security. If enabled, an attacker could abuse this flaw to obtain the user's credentials from the log files.",
    "poc": {
        "reference": [
            "https://access.redhat.com/errata/RHSA-2020:0727"
        ],
        "github": [
            "No PoCs found on GitHub currently."
        ]
    }
}